wire-grpc-client/build.gradle

```diff
@@ -19,4 +19,9 @@ dependencies {
   api deps.okio.jvm
   api deps.okhttp
   api deps.kotlin.coroutines.core
+  if (JavaVersion.current().isJava9Compatible()) {
+    // Workaround for @javax.annotation.Generated
+    // see: https://github.com/grpc/grpc-java/issues/3633
+    api 'javax.annotation:javax.annotation-api:1.3.1'
```

Comment:
I thought you said this only affects tests. Why is it being added as a transitive dependency of this artifact?

Reply:
Because I totally pooped on myself. It's updated.